src/content/docs/it/recipes.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Altre soluzioni
+description: Esempi guidati su come aggiungere nuove funzionalità al tuo progetto Astro.
+i18nReady: true
+---
+
+import RecipesNav from '~/components/RecipesNav.astro';
+
+Esempi guidati su come aggiungere nuove funzionalità al tuo progetto Astro.
+
+## Soluzioni ufficiali
+
+<RecipesNav />
+
+## Soluzioni dalla community
+
+Se hai una soluzione che vuoi condividere, consulta [le nostre linee guida per contribuire](https://github.com/withastro/docs/blob/main/contributor-guides/submitting-a-recipe.md)  
+
+- [Importare immagini utilizzando un nome file dinamico](https://vaihe.com/blog/astro/astro-dynamic-image-prop/)
+- [Aggiungere transizioni di pagina animate con Swup](https://navillus.dev/blog/astro-plus-swup)
+- [Utilizzare UnoCSS in Astro](https://www.elian.codes/blog/23-02-11-implementing-unocss-in-astro/)
+- [Costruire una tabella dei contenuti utilizzando le intestazioni dei file markdown di Astro](https://kld.dev/building-table-of-contents/)
+- [Creare un plugin Remark per rimuovere i "righini" dai tuoi file Markdown](https://eatmon.co/blog/remove-runts-markdown/)
```

Comment:
"Righini" (a word completely unknown to me until today) is the translation of "runts".
See: 
https://denisesarrecchia.org/2020/02/20/orfane-vedove-e-righini-gli-errori-grafici-piu-comuni/ 
https://www.treccani.it/vocabolario/righino/